sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -705,7 +705,7 @@ Azure::Core::_internal::UniqueHandle<HINTERNET> WinHttpTransport::CreateSessionH
 // set successfully, we shouldn't fail the request and continue as if the options don't exist.
 // Therefore, we just ignore the error and move on.
 #ifdef WINHTTP_OPTION_TCP_FAST_OPEN
-  BOOL tcp_fast_open = TRUE;
+  BOOL tcp_fast_open = FALSE;
```

Comment:
I would add a comment explaining why the comment says that it's enabling the fast open option but the code says it's disabling the fast open option.

Alternatively, you could change the #ifdef at line 707 to:

```cpp
#if defined(WINHTTP_OPTION_TCP_FAST_OPEN) && FALSE
```

which completely removes the code.

Without a comment, someone is going to come along next year and send an email to the C++ SDK team asking "Does anyone know why this code claims to enable fast open but it doesn't actually do that?" :)

Reply:
Excellent point.  I'll make that change as I figure we will have this code in here for some time will we wait on the fix.